portals/new_admin/portals/tsconfig.json

```diff
@@ -0,0 +1,13 @@
+{
+    "compilerOptions": {
+        "outDir": "./dist/",
+        "sourceMap": true,
+        "noImplicitAny": true,
+        "module": "es6",
+        "target": "es5",
+        "jsx": "react",
+        "allowJs": true,
+        "moduleResolution": "node",
+        "allowSyntheticDefaultImports": true
+    }
+}
```

Comment:
Shall we fix the EOF newline missing issue? Please fix it in other places too

Reply:
+1